ndc-oslo-2016/videos/ndc-oslo-2016-mutation.json

```diff
@@ -0,0 +1,32 @@
+{
+  "alias": "video/python-an-amazing-second-language-for-dotnet-developers",
+  "category": "NDC Oslo 2016",
+  "copyright_text": "NDC Conferences",
+  "description": "The modern software development landscape is a terrain of many platforms and technologies. Gone are the days where simply knowing one technology really really well is sufficient to stay on the cutting edge. We are expected to write web-based applications, mobile applications, cross-platform applications and so on. Even as we know we should learn more and branch out, that choice is increasingly difficult as the technology options explode.\nThis talk will give you one very solid choice that is familiar to .NET developers: Python. It may seem like a very different language and ecosystem when compared to .NET but just below the surface, there are many more similarities than there are differences. For .NET's Entity Framework, Python has SQLAlchemy. For .NET's ASP.NET MVC, Python has The Pyramid Web Framework. The list goes on and on.\nCome learn why your team should learn Python as a complementary language to .NET.",
+  "duration": 3628,
+  "id": 5982,
+  "language": "eng",
+  "quality_notes": "",
+  "recorded": "2016-06-21",
+  "slug": "python-an-amazing-second-language-for-dotnet-developers",
+  "speakers": [
+    "Michael Kennedy"
+  ],
+  "summary": "The modern software development landscape is a terrain of many platforms and technologies. Gone are the days where simply knowing one technology really really well is sufficient to stay on the cutting edge. We are expected to write web-based applications, mobile applications, cross-platform applications and so on. Even as we know we should learn more and branch out, that choice is increasingly difficult as the technology options explode.\nThis talk will give you one very solid choice that is familiar to .NET developers: Python. It may seem like a very different language and ecosystem when compared to .NET but just below the surface, there are many more similarities than there are differences. For .NET's Entity Framework, Python has SQLAlchemy. For .NET's ASP.NET MVC, Python has The Pyramid Web Framework. The list goes on and on.\nCome learn why your team should learn Python as a complementary language to .NET.",
+  "tags": [
+    "language",
+    "polyglot",
+    "dotnet",
+    "ndc",
+    "ndcoslo",
+    "ndcoslo2016"
+  ],
+  "thumbnail_url": null,
```

Comment:
Do you have a thumbnail for this video?

Something like: `https://i.vimeocdn.com/video/576725146.jpg?mw=960&mh=540`